lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -135,6 +135,21 @@ std::vector<std::string> GetStrings(const llvm::json::Object *obj,
   return strs;
 }
 
+static std::string GetDescriptionTrimmed(lldb::SBValue &value) {
+  lldb::SBStream stream;
+  if (!value.GetDescription(stream)) {
+    return "";
+  }
```

Comment:
remove unnecessary braces

Reply:
Removed the braces (and well simplified this function).